src/assets/i18n/de.json

```diff
@@ -150,6 +150,19 @@
 		"editMemberInstitute": "Bearbeite die Mitglieder der Institution.",
 		"membersHaveAccess": "Diese Mitglieder haben Zugriff auf die Institution.",
 		"addMember": "Mitglied hinzufügen",
+		"addMember_text": "Neue Mitarbeiter müssen ein OEB-Konto anlegen und verifizieren, bevor sie zu einem Emittenten hinzugefügt werden.",
```

Comment:
Instead of ``` Emittenten ``` please use ```Issuer``` (we use Issuer in english and german, it was a request by mycelia)

Reply:
Done!